submissions/fabunmiTolulope/css/product-page.css

```diff
@@ -0,0 +1,437 @@
+*{
+	box-sizing: border-box;
+	margin: auto;
+}
+body{
+	font-size: 16px;
+	font-family: "Georgia", Times New Roman;
+}
+header{
+	flex: 1 100%;
+	background-color: #B7EEB7;
+	height: 70px;
+}
+header *{
+	color: #FF530E;
+}
+header .title{
+	float: left;
+	height: inherit;
+	padding: 10px 30px;
+}
+header ul li{
+	float: right;
+	height: inherit;
+	padding: 20px;
+	list-style: none;
+	display: inline-block;
+}
+a{
+	text-decoration: none;
+	color: #FF530E;
+}
+a:hover{
+	text-decoration: underline;
+	color: rgb(200, 0, 0);
+	cursor: pointer;
+}
+.cavendish{
+	display: flex;
+	flex-direction: row;
+  	justify-content: space-between;
+}
+.cavendish img{
+	float: left;
+	height: 180px;
+    width: 250px;
+    border: 1px solid gray;
+    margin: 30px 35px 0 60px;
+    padding: 15px;
+}
+.cost{
+	font-size: 1.1em;
+	font-weight: bold;
+	margin-left: 40px;
+}
+.cost h3{
+	font-size: 1.1em;
+	}
+#quantity{
+	font-size: 14px;
+	padding: 5px;
+	width: 60px;
+}
+#btn{
+	font-weight: bold;
+    padding: 10px;
+    background-color: #B7EEB7;
+    font-size: 15px;
+    border-radius: 5px;
+}
+#btn:disabled{
+	background-color: gray;
+}
+#btn:hover{
+	background-color: gray;
+    color: #FF530E;
+    cursor: pointer;
+    }
+.description{
+	clear: both;
+	display: block;
+	flex: 1 100%;
+	width: 100%;
+}
+.description h3{
+	font-size: 1.4em;
+	line-height: 40px;
+	text-align: left;
+}
+.similar{
+  	text-align: center;
+    background-color: gray;
+    padding: 10px;
+}
+.similar h3{
+	font-size: 1.2em;
+}
+.similar ul li{
+	background-color: white;
+	border: 1px solid gray;
+	display: inline-grid;
+	padding: 20px;
+	margin: 20px 80px;
+}
+.similar img{
+	height: 180px;
+	width: 220px;
+}
+.similar figcaption{
+	font-weight: bold;
+	text-align: center;
+}
+footer{
+	background-color: #B7EEB7;
+	color: #FF530E;
+	display: flex;
+	flex: 1 100%;
+	width: 100%;
+	align-content: center;
+}
+footer ul li{
+	text-align: left;
+	padding: 5px;
+	list-style: none;
+}
+.footerlist{
+	margin-right: 250px;
+}
+.footer-signup{
+		margin: auto;
+}
+#footer-email{
+	margin-top: 10px;
+	width: 250px;
+	height: 30px;
+}
+#footer-submit{
+	margin-top: 10px;
+	font-size: 15px;
+	width: 80px;
+	height: 30px;
+	background-color: #B7EEB7;
+	color: #FF530E;
+}
+.footer-connect{
+	padding: 10px;
+	margin-left: 250px;
+}
+.footer-connect .img{
+	height: 40px;
```

Comment:
Set just only one of either height or width not both @ToluEnny24

Reply:
i have modified across the whole doument